pkg/relayer/proxy/proxy_test.go

```diff
@@ -516,6 +518,66 @@ func TestRelayerProxy_Relays(t *testing.T) {
 			expectedErrCode: -32000,
 			expectedErrMsg:  "session expired", // Relay rejected by the supplier
 		},
+		// How do I do this? :D
```

Comment:
@red-0ne I'm not familiar with relay miner testing suite, what's the best way to test gzip compressed response?

Reply:
Nvm, we decided test will be added in a subsequent pr.